health-services/project-factory/src/server/utils/campaignUtils.ts

```
@@ -1259,28 +1234,27 @@
 async function enrichAndPersistProjectCampaignForFirst(
   request: any,
   actionInUrl: any,
-  firstPersist: boolean = false,
-  localizationMap?: any
+  firstPersist: boolean = false
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)